guard/src/commands/validate.rs

```diff
@@ -852,22 +858,25 @@ fn evaluate_against_data_input<'r>(
     Ok(overall)
 }
 
-fn get_path_aware_value_from_data(content: &String) -> Result<PathAwareValue> {
```

Comment:
Should we change this method to return `DataFile` struct and extract the members out wherever we only need like `content` or `path_value`?

Reply:
I like the suggestion, done.